### PR TITLE
Make fireaxe & shotgun cabinets destructible

### LINKED ISSUE #25525: Fireaxe cabinet isnt destructible
has Damageable component but not desctructible

### PR DESCRIPTION
**Pull request by Gyrandola**
## About the PR

Makes fireaxe and shotgun cabinets destructible.

## Why / Balance

Fixes #25525. 

## Technical details

Simply added the component to the fireaxe cabinet base entity.
Glass breaking sounds regardless of state.
Damage thresholds:
min: 200
max: 300 (The axe will be destroyed as well).

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

Screenshot 2024-03-10 123916

Screenshot 2024-03-10 124912

## Breaking changes

**Changelog**

:cl:
- add: Fireaxe and Shotgun cabinets can now be destroyed.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Wallmounts/fireaxe_cabinet.yml [hunk 1]
```diff
@@ -3,9 +3,21 @@
   name: fire axe cabinet
   description: There is a small label that reads "For Emergency use only" along with details for safe use of the axe. As if.
   components:
-  - type: Damageable # adding destructible causes the entity inside to be deleted when the cabinet is destroyed :(
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Structures/Wallmounts/fireaxe_cabinet.yml:6]: Does the axe now appear when cabinet is destroyed?

Comment (Gyrandola) [reply Resources/Prototypes/Entities/Structures/Wallmounts/fireaxe_cabinet.yml:6]: From what I've seen, yes. I posted screenshots.

**Comment by TurboTrackerss14** [comment]
I would also strongly suggest adding
1) a minimum damage threshhold inline with lockers and/or reinforced windoors. 
2) shotgun wall cabinets too

But also, invulnerable fireaxe locker is funny keep it.

**Comment by Gyrandola** [comment]
> But also, invulnerable fireaxe locker is funny keep it.

If it's because of ss13 I get it, but one could instead just make it really tough to break.

**Comment by K-Dynamic** [comment]
I don't really get why 100 dmg should destroy both cabinet and fireaxe, and 75 damage would mean just bashing a crowbar about 7 times (assuming structural dmg counted) to break cabinet open

**Comment by Gyrandola** [comment]
> I don't really get why 100 dmg should destroy both cabinet and fireaxe, and 75 damage would mean just bashing a crowbar about 7 times (assuming structural dmg counted) to break cabinet open
I looked at all the other cabinets (defib, filing, extinguisher) which instead destroy the contents at only 80, so i figured a fireaxe would be a little tougher. It also makes it so the axe wont survive a direct RPG shot.

**Comment by K-Dynamic** [comment]
> I looked at all the other cabinets (defib, filing, extinguisher) which instead destroy the contents at only 80, so i figured a fireaxe would be a little tougher. It also makes it so the axe wont survive a direct RPG shot or something of the kind.

Yeah but consider how quickly you can break those other cabinets, then consider how dangerous a stolen fireaxe can be

Incidentally, how much dmg can the shotgun cabinet take?

**Comment by Gyrandola** [comment]
> > I looked at all the other cabinets (defib, filing, extinguisher) which instead destroy the contents at only 80, so i figured a fireaxe would be a little tougher. It also makes it so the axe wont survive a direct RPG shot or something of the kind.
> 
> Yeah but consider how quickly you can break those other cabinets, then consider how dangerous a stolen fireaxe can be
> 
> Incidentally, how much dmg can the shotgun cabinet take?

The shotgun cab is a child of fireaxe cab so the damage would be the same now. I recognize its a problem. I'll look into tweaking the numbers later today.

**Pull request merged**